### PR TITLE
Add page on configuring cloud settings

### PR DESCRIPTION
**Pull request by justindeguzman**

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit ed0c8844da2ccfe35288b327b6f77253b0e9ee9b with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1421/ed0c8844da2ccfe35288b327b6f77253b0e9ee9b/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

File: docs/en/cloud/manage/settings.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+sidebar_label: Configuring Settings
+slug: /en/manage/settings
+---
+
+# Configuring Settings
+
+To specify settings for your ClickHouse Cloud service for a specific [user](/docs/en/operations/access-rights#user-account-management) or [role](/docs/en/operations/access-rights#role-management), you must use [SQL-driven Settings Profiles](/docs/en/operations/access-rights#settings-profiles-management). Applying Settings Profiles ensures that the settings you configure persist, even when your services stop, idle, and upgrade. To learn more about Settings Profiles, please see [this page](/docs/en/operations/settings/settings-profiles.md).
+ 
+Please note that XML-based Settings Profiles and [configuration files](/docs/en/operations/configuration-files.md) are currently not supported for ClickHouse Cloud.
```
Comment (melvynator) [inline docs/en/cloud/manage/settings.md:10]: Here it's mentioned `currently not supported`. I don't know if we intend to support it in the future. Maybe we should say `are not supported`

**Review by melvynator — APPROVED** [review]
LGTM

File: docs/en/cloud/manage/settings.md (hunk 1 above)
Comment (tbragin) [reply docs/en/cloud/manage/settings.md:10]: I am ok with "currently" - our plans can always change even if something is not on the immediate roadmap, so it is not inaccurate.

**Pull request merged**